backend/src/main/java/edu/pl/masi/yellow/controller/api/v1/manage/TestIO.java

```diff
@@ -0,0 +1,64 @@
+package edu.pl.masi.yellow.controller.api.v1.manage;
+
+import edu.pl.masi.yellow.entity.UserEntity;
+import edu.pl.masi.yellow.manager.TestFormatterManager;
+import edu.pl.masi.yellow.manager.UserManager;
+import edu.pl.masi.yellow.model.LoginToken;
+import edu.pl.masi.yellow.model.response.GenericResponse;
+import edu.pl.masi.yellow.utils.exceptions.ForbiddenException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@RestController
+public class TestIO {
+    private UserManager userManager;
+    private TestFormatterManager testIOFormatter;
+
+    @RequestMapping(path = "/api/v1/tests/io/csv/{testId}", method = RequestMethod.GET)
+    public void generateTestCSV(@RequestHeader(name = "Auth-Token", required = false) LoginToken authToken,
```

Comment:
Correct me if I'm wrong but shouldn't we mark this methods as throws ForbiddenException or something like this?
The same in every method that throws ForbiddenException().